### ISSUE TITLE
[Enhancement]: Add support for aria-label on va-additional-info

### ISSUE THREAD

**Issue by darbyemadewell** [department-of-veterans-affairs/vets-design-system-documentation#6028]
### Verify the following

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.
- [x] To my knowledge this is not an accessibility issue. If this is an accessibility issue, use the accessibility issue template

### Type of issue

Enhancement

### Select your product area

Benefits Portfolio

### Team Name

Accredited Representative Crew

### Team label

ar-crew

### Point of contact

@darbyemadewell

### What happened

One of our tools ([Find a Representative](https://staging.va.gov/get-help-from-accredited-representative/find-rep/)) has search results that include the `va-additional-info` component on each card. The ideal UX path is to have duplicate visual labels and distinguish each link to screen reader users with an `aria-label`. The `va-additional-info` component currently does not have a way to pass an `aria-label` to the internal `a` tag.

### What I expected to happen

Ideally, we would like to be able to pass an `aria-label` into the `va-additional-info` component's internal `a` tag, as shown here:

<img width="1512" height="945" alt="Image" src="https://github.com/user-attachments/assets/f9240332-912d-452f-a347-3047ae5dfbe3" />

### URL where issue occurs

https://staging.va.gov/get-help-from-accredited-representative/find-rep/

### Steps to reproduce

1. Try adding an `aria-label` to the `va-additional-info` component.
2. See that the `aria-label` can't be passed down to the internal anchor tag.

### Urgency

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [x] This bug is not blocking any work
- [ ] Other
